microhapdb/tests/test_cli.py

```diff
@@ -288,7 +289,7 @@ def test_main_marker_view_bad():
     ('--population=SA000009J', '--marker=mh13KK-218', None, 15),
     (None, '--marker=mh13KK-218', '--allele=C,T,C,T', 97),
     (None, '--marker=mh14PK-72639', None, 217),
-    (None, None, None, 126289)
+    (None, None, None, 130420)
```

Comment:
Updated total length

Reply:
Yeah, for tests like this, it's more of a consistency check than a correctness check. I haven't put much work into carefully accounting for every record in the frequency table.

Reply:
Although, 130420 - 126289 = 4131, which is reassuring to see below.